Epilepsy/README.md

```diff
@@ -0,0 +1,10 @@
+# Epilepsy
+
+This folder holds our experiment where the goal is to extract known Disease-Gene relationships in respect to Epilepsy. The entuire task has been divided into three parts specified below.
+
+## The Scripts
+1. Data Processing [[Data Parser File](Epilepsy Data Parser.ipynb)]: This file does the heavy lifting in regards to tagging/parsing the given corpora. Also, this extracts candidates from the tagged text as well.
```

Comment:
Let's use hyphens rather than spaces in file names. Also all lowercase is often easiest: `epilepsy-data-parser.ipynb`.